src/mediatype-icon.ts

```diff
@@ -44,21 +49,44 @@ export class MediatypeIcon extends LitElement {
     film: 'Film',
     image: 'Image',
     movies: 'Movie',
+    radio: 'Radio',
     software: 'Software',
     texts: 'Text',
     tv: 'TV',
     video: 'Video',
     web: 'Web',
   };
 
+  private getDisplayMediaType = () => {
+    const tvIdentifier = ['tvnews', 'tvarchive', 'television'];
+    const radioIdentifier = ['radio', 'radioprogram'];
+
+    if (
+      this.mediatype === 'movies' &&
+      this.collection?.some(id => tvIdentifier.indexOf(id) >= 0)
+    ) {
+      return 'tv';
+    }
+    if (
+      this.mediatype === 'audio' &&
+      this.collection?.some(id => radioIdentifier.indexOf(id) >= 0)
+    ) {
+      return 'radio';
+    }
+    return this.mediatype;
+  };
+
   render() {
     if (!this.mediatype) {
       return html``;
     }
+
+    const displayMediatype = this.getDisplayMediaType() || '';
```

Comment:
Does `getDisplayMediaType()` return an optional?

Reply:
i'm getting:
```
const displayMediatype: string | undefined
Type 'undefined' cannot be used as an index type.ts(2538)
```

if i don't add `|| ''` in the end